Example/CMHealth.xcodeproj/project.pbxproj

```diff
@@ -218,30 +229,19 @@
 			name = "Podspec Metadata";
 			sourceTree = "<group>";
 		};
-		ED91E6E2C6596827EBA65304 /* Pods */ = {
-			isa = PBXGroup;
-			children = (
-				812C4B8738B6F9701D594161 /* Pods-CMHealth_Example.debug.xcconfig */,
-				2C7A9EE1CC8E5EB244D173F4 /* Pods-CMHealth_Example.release.xcconfig */,
-				974DEEB5A4116D10FF749933 /* Pods-CMHealth_Tests.debug.xcconfig */,
-				FEB06221139C06DAA962E0A6 /* Pods-CMHealth_Tests.release.xcconfig */,
-			);
-			name = Pods;
-			sourceTree = "<group>";
-		};
 /* End PBXGroup section */
 
 /* Begin PBXNativeTarget section */
 		6003F589195388D20070C39A /* CMHealth_Example */ = {
 			isa = PBXNativeTarget;
 			buildConfigurationList = 6003F5BF195388D20070C39A /* Build configuration list for PBXNativeTarget "CMHealth_Example" */;
 			buildPhases = (
-				F361B9EB3956C2238FDC255F /* Check Pods Manifest.lock */,
+				16129520D8CCEAFB0B4B6C62 /* 📦 Check Pods Manifest.lock */,
```

Comment:
Ha! I know its silly- but actually it is CocoaPods that is putting those there, not me. They use it to denote settings placed by CocoaPods in the new version.